test/cypress/e2e/multi-datasources/multi_datasources_enabled.spec.js

```diff
@@ -0,0 +1,244 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+const externalTitle = '9202';
```

Comment:
Where does this name come from?

Reply:
It is simply a human readable label for the datasource - I moved this to the cypress config file so if you are running the tests on your local you can choose a different name/based on what you want.